config/features.rb

```diff
@@ -19,4 +19,8 @@
   feature :import_mode,
           default: false,
           description: "Import mode - Background jobs are run on specially named queues"
+
+  feature :task_master,
+          default: false,
```

Comment:
What do you think about setting this based upon the ENV?  This way we don't have to remember to turn it on for every tenant.
```suggestion
          default: ENV["PUBSUB_SERVICEACCOUNT_KEY"].present?,
```